mevislab.github.io/themes/MeVisLab/layouts/shortcodes/tags.html

```diff
@@ -0,0 +1,307 @@
+                                if (mykey == 'title') mytitle_matches = mytitle_matches + mycount;
+                                if ((mykey == 'tags') && (mytags != '')) mytags_matches = mytags_matches + mycount;
+                                if (mykey == 'summary') mysummary_matches = mysummary_matches + mycount;
+                                if (mykey == 'content') mycontent_matches = mycontent_matches + mycount; 
+                            }           
+                        }
+                        mymatched++;
+                    }
+                }
+                mydata[i]['mymatches_calc'] = mymatches_calc;
+                if ((debug != '') && (debug != null)) {
+                    mydata[i]['mymatches'] = mymatches;
+                    mydata[i]['mytitle_matches'] = mytitle_matches;
+                    mydata[i]['mytags_matches'] = mytags_matches;
+                    mydata[i]['mysummary_matches'] = mysummary_matches;
+                    mydata[i]['mycontent_matches'] = mycontent_matches;
+                }
+                if (mymatched == mywords.length) {
+                    myresults.push(mydata[i]);
+                }
+            } 
+
+            // results
+            if (myresults.length >= 1) {
+                myadd_searchlink = "";
+                myresults.sort(function(a, b) {
+                if (static_params['sort_date'] == 'DESC') {
+                    return b.mymatches_calc - a.mymatches_calc || b.mydate - a.mydate;
+                } else {
+                    return b.mymatches_calc - a.mymatches_calc || a.mydate - b.mydate;
+                }
+                }); 
+
+                var myresults_header = '';
+                if (myresults.length > 1) {
+                    myresults_header = static_params['res_more_items'];
+                } else {
+                    myresults_header = static_params['res_one_item'];
+                } 
+                myresults_header = myresults_header.replace("[CNT]", myresults.length);
+                if ((myadd_searchlink) && (myadd_searchlink != '')) {
+                    myresults_header += myadd_searchlink;
+                }
+
+                var mytag_top = static_params['res_out_top'];
+                var mytag_bottom = static_params['res_out_bottom'];
+
+                var myresults_content = '';
+                for (var i = 0; i < myresults.length; i++) {
+                    var mytitle   = myresults[i].title;
+                    var mysummary = myresults[i].summary;
+                    var mydate    = myresults[i].date;
+                    var myurl     = myresults[i].url;
+                    var mytags    = myresults[i].tags;
+                    var mysection = myresults[i].section;
+                    var myextern  = myresults[i].extern;
+
+                    // only for debug mode
+                    var myinfos = '';
+                    if ((debug != '') && (debug != null)) {
```

Comment:
Just `if(debug)` ?

Reply:
i prefer to leave this as is, because its working :)